<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createOrder</class>
 <widget class="QWidget" name="createOrder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="calculateShippingBtn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>330</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate Shipping</string>
   </property>
  </widget>
  <widget class="QLabel" name="shippingCostLabel">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>320</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Shipping cost = </string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>351</width>
     <height>111</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="title">
    <string>Additional details</string>
   </property>
   <widget class="QLabel" name="durabilityLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Durability</string>
    </property>
   </widget>
   <widget class="QComboBox" name="durabilityCombo">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>Not fragile</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fragile</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="durabilityLabel_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Required temperature</string>
    </property>
   </widget>
   <widget class="QComboBox" name="tempCombo">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Room temperature</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Chilled (3°C)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Frozen (-20°C)</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QPushButton" name="requestOrderBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>410</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Request order</string>
   </property>
  </widget>
  <widget class="QLabel" name="lorryTypeLabel">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>350</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lorry type = </string>
   </property>
  </widget>
  <widget class="QGroupBox" name="locationBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>371</width>
     <height>151</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="title">
    <string>Journey details</string>
   </property>
   <widget class="QLineEdit" name="destinationField">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="postcodeField">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="distanceLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="text">
     <string>Distance from source to destination (km)</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="distanceField">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="postcodeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Source Postcode</string>
    </property>
   </widget>
   <widget class="QLabel" name="destinationLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Destination Postcode</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="cargoDetailsBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>301</width>
     <height>191</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="title">
    <string>Cargo dimensions</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="weightLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Weight (kg)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="weightField">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="widthLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Width (m)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="widthField">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="heightLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Height (m)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="heightField">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lengthLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Length (m)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lengthField">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
